### PR TITLE
Document develop-9.0 as recommended docs fix target branch

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

**Comment by montegoulding** [comment]
@livecode-vulcan review ok f84c826

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok f84c826942481d0f64406d2be0609b98eb88c18c

**Comment by livecode-vulcan** [comment]
😎 test success f84c826942481d0f64406d2be0609b98eb88c18c

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/139)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/131)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/131)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/131)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/131)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1653)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/35)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/35)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1683)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1676)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1673)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1709)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1650)

File: docs/contributing_to_docs.md [hunk 1]
```diff
@@ -155,19 +155,19 @@ website, without having to download any software or use the command line.
 
 ### Making your dictionary change
 
-**Navigate to the file you want to modify in the [dictionary folder of the develop-8.2 branch](https://github.com/livecode/livecode/blob/develop-8.2/docs/dictionary).**
+**Navigate to the file you want to modify in the [dictionary folder of the develop-9.0 branch](https://github.com/livecode/livecode/blob/develop-9.0/docs/dictionary).**
```
Comment (seaniepie) [inline docs/contributing_to_docs.md:158]: Thanks for all these, Panos. Very useful

**Pull request merged**